.env.sample

```diff
@@ -20,6 +20,7 @@ EMAIL_REVIEW=
 EMAIL_SENDER=
 EMAIL_SHIP=
 EXPERIMENTS_PAGINATE_BY=50
+EXPERIMENTER_INSTANCE=local
```

Comment:
Can't this just use `HOSTNAME` [https://github.com/mozilla/experimenter/blob/main/.env.sample#L26](here)?

Reply:
oh! is hostname for stage: "stage" and prod: "prod"?

Reply:
no it's the full url, do you need something that's just 'local' 'stage' 'prod'? how does ndt use that?

Reply:
yup, devtools urls follow this pattern ext+normandy://<local, dev, stage, prod>/recipes

Reply:
does ndt use that to figure out which experimenter url to use?

Reply:
errrm do you mean does experimenter uses it to figure out which ndt url to use?

Reply:
No like why is it passed to NDT? How does it use it?